### PR TITLE
Replace ioutil package with relevant methods

### PR DESCRIPTION
**Pull request by DiptoChakrabarty**
This PR replaces the io/ioutil deprecated package with os and io packages methods. This works on issue #2994

### CODE REVIEW

**Review by Liuhaai — APPROVED** [review]

**Comment by Liuhaai** [comment]
thx for contribution

**Review by millken — APPROVED** [review]

File: ioctl/newcmd/account/accountdelete.go [hunk 1]
```diff
@@ -110,6 +110,23 @@ func NewAccountDelete(c ioctl.Client) *cobra.Command {
 						filePath = v.URL.Path
 						break
 					}
+
+					aliases := make(map[string]string)
```
Comment (Liuhaai) [inline ioctl/newcmd/account/accountdelete.go:114]: Is this change related to the issue?

**Review by Liuhaai — CHANGES_REQUESTED** [review]

File: ioctl/newcmd/account/accountdelete.go (hunk 1 above)
Comment (DiptoChakrabarty) [reply ioctl/newcmd/account/accountdelete.go:114]: no I think it was part of the code while rebasing I will fix it

**Review by Liuhaai — APPROVED** [review]

**Review by Scheusal13 — COMMENTED** [review]
io168vpa7fn9xk3jerkjf4mjgdmcrg3hv8af9qncm
